frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -59,17 +66,22 @@ const ApartmentCard = ({ buildingData, numReviews, company }: Props): ReactEleme
   const { name, photos } = buildingData;
   const img = photos.length > 0 ? photos[0] : ApartmentImg;
   const classes = useStyles();
```

Comment:
You can destructure `classes` like this
```
const {img, imgMobile, aptNameTxt, etc} = useStyles();
```
so that you don't have to keep writing `className={classes.something}`